dependencies.yaml

```diff
@@ -29,6 +30,12 @@ files:
     includes:
       - checks
       - py_version
+  docs:
+    output: none
+    includes:
+      - cudatoolkit
+      - docs_build
```

Comment:
This comment is relevant: https://github.com/rapidsai/rmm/pull/1189#discussion_r1073685857